sale_order_type/models/sale.py

```diff
@@ -12,6 +12,7 @@ class SaleOrder(models.Model):
         string="Type",
         compute="_compute_sale_type_id",
         store=True,
+        domain="[('company_id', 'in', [False, company_id])]",
```

Comment:
Why not using the new `check_company=True` ?

https://www.odoo.com/documentation/13.0/howtos/company.html#multi-company-consistency